### ISSUE TITLE
NTR: Muscle biopsy: carnitine palmitoyltransferase low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9065]
New term request Muscle biopsy: carnitine palmitoyltransferase low (f431): CPT II  UMLS: 2874243

**Comment by pnrobinson** [comment]
@MickeySegal

I think the existing term may actually be about muscle findings, I will investigate:
[Reduced carnitine O-palmitoyltransferase level HP:0012380](https://hpo.jax.org/app/browse/term/HP:0012380)

**Comment by MickeySegal** [comment]
That sounds right.  I'll use that HPO code.

**Comment by pnrobinson** [comment]
@MickeySegal  -- please check whether the following is correct for your use of the code:

looking at the annotations and the definition,  I think the code meant the isozyme 2. I have revised to

id: [HP:0012380](https://hpo.jax.org/app/browse/term/HP:0012380)
label Reduced tissue carnitine O-palmitoyltransferase 2 activity
def: Reduced carnitine O-palmitoyltransferase (CPT II; EC 2.3.1.21) activity or level, leading to a reduced activity of the reaction: palmitoyl-CoA + L-carnitine = CoA + L-palmitoylcarnitine. (CPT II activity can be measured in tissues including fibroblasts, lymphoblasts, and skeletal muscle.

**Comment by MickeySegal** [comment]
The testing of CPT activity seems like it is measuring the whole CPT complex (CPT I and CPT II):
https://www.ncbi.nlm.nih.gov/books/NBK1253/
"Tests of total CPT enzyme activity (both CPT I and CPT II) rely on the basic reaction: palmitoyl-CoA + carnitine ↔ palmitoylcarnitine + CoA. Activity of CPT II represents only 20%-40% of total CPT activity. Measured enzyme activity is dependent on assay conditions, which have not been standardized, making comparisons of published data from different laboratories difficult:

- The "radio isotope exchange assay" described by Norum [1964] is still widely used.

- The "isotope forward assay" measures total CPT enzyme activity (CPT I and CPT II) by the incorporation of radio-labeled carnitine into palmitoylcarnitine [Zierz & Engel 1985]. Total CPT enzyme activity is normal in both affected individuals and controls. In this assay, CPT II enzyme activity is measured as the fraction that is not inhibited by malonyl-CoA."

So it seems better to have the enzyme activity be for carnitine palmitoyltransferase complex as a whole.
